src/main/java/org/embulk/output/kintone/reducer/ReduceType.java

```diff
@@ -141,7 +141,7 @@ public static MapValue value(Value value) {
 
   public static MapValue value(Long id, MapValue value, MapValue sortValue) {
     ValueFactory.MapBuilder builder = ValueFactory.newMapBuilder();
-    builder.put(ID, id == null ? NIL : ValueFactory.newInteger(id));
+    builder.put(ID, id == null ? NIL : ValueFactory.newString(id.toString()));
```

Comment:
SUBTABLE row id is Long in [kintone-java-client](https://github.com/kintone/kintone-java-client/blob/master/src/main/java/com/kintone/client/model/record/TableRow.java#L20), but it is string in [API document](https://cybozu.dev/ja/kintone/docs/overview/field-types/#field-type-update).